tests/alerts/test_ssh_key.py

```diff
@@ -69,15 +69,6 @@ class TestSSHKey(AlertTestSuite):
         )
     )
 
-    event = AlertTestSuite.create_event(default_event)
```

Comment:
Since we replaced '_type' with category in the alert, we should have a negative test with a bad category.

Reply:
Good call, I have submitted a commit to resolve.